src/services/hooks.rs

```diff
@@ -1,23 +1,27 @@
-use crate::utils::context::Context;
+use crate::utils::context::ContextCache;
 use crate::models::backend::webhooks::{Payload, PayloadDetails};
 use anyhow::Result;
+use crate::utils::cache::Cache;
 
-pub fn invalidate_caches(context: &Context, payload: &Payload) -> Result<()> {
+pub fn invalidate_caches(context: &impl ContextCache, payload: &Payload) -> Result<()> {
     let cache = context.cache();
     cache.invalidate_pattern(&format!("*{}*", &payload.address));
+    println!("{:#?}", &payload);
+
     payload.details.as_ref().map(|d| {
         match d {
             PayloadDetails::NewConfirmation(data) => {
                 cache.invalidate_pattern(&format!("*{}*", data.safe_tx_hash));
-            },
```

Comment:
the `,` is not needed?

Reply:
I think the auto formatter from Intellij decided it wasn't. It still runs the tests with no hiccups.